server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+[id="proc-creating-oidc-client_{context}"]
+==== Creating an Open ID Connect Client
+
```

Comment:
We need at least one sentence here about the purpose of the procedure, which is essentially to secure an exisitng application as an Open ID connect client.

We also need a heading
.Procedure